From 50a425dc4eeb5f184e67b4545bcaeda51698b3a7 Mon Sep 17 00:00:00 2001
From: Jonas Bernoulli <jonas@bernoul.li>
Date: Tue, 30 Nov 2010 22:20:25 +0100
Subject: [PATCH 2/4] dont install bup-web

---
 Makefile |    9 +--------
 1 files changed, 1 insertions(+), 8 deletions(-)

diff --git a/Makefile b/Makefile
index 4647f59..4816395 100644
--- a/Makefile
+++ b/Makefile
@@ -22,8 +22,7 @@ BINDIR=$(DESTDIR)/usr/bin
 LIBDIR=$(DESTDIR)/usr/lib/bup
 install: all
 	$(INSTALL) -d $(MANDIR)/man1 $(DOCDIR) $(BINDIR) \
-		$(LIBDIR)/bup $(LIBDIR)/cmd \
-		$(LIBDIR)/web $(LIBDIR)/web/static
+		$(LIBDIR)/bup $(LIBDIR)/cmd
 	[ ! -e Documentation/.docs-available ] || \
 	  $(INSTALL) -m 0644 \
 		Documentation/*.1 \
@@ -42,12 +41,6 @@ install: all
 	$(INSTALL) -m 0755 \
 		lib/bup/*$(SOEXT) \
 		$(LIBDIR)/bup
-	$(INSTALL) -m 0644 \
-		lib/web/static/* \
-		$(LIBDIR)/web/static/
-	$(INSTALL) -m 0644 \
-		lib/web/*.html \
-		$(LIBDIR)/web/
 %/all:
 	$(MAKE) -C $* all
 
-- 
1.7.4.1

